CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 
 Unreleased
 ==========
+
+1.6.3 (2022-09-21)
```

Comment:
A minor release here would generally imply it is a bug fix.

Reply:
Yeah I thought this too, it just felt like such a small feature that is didnt _feel_ like it warranted a bump to 1.7.0. But perhaps I should for consistency? I'm happy with either